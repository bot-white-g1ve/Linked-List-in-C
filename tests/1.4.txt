NEW 1
a
VIEW 0
REMOVE 0
VIEW 0
VIEW ALL