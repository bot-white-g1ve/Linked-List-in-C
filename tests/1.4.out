List 0: a
a
List 0 has been removed.

Number of lists: 0
INVALID COMMAND: VIEW
Number of lists: 0
